### PR TITLE
MF-763 Enhance vitals and biometrics chart to match designs

### PR DESCRIPTION
**Pull request by maureennduta**
## Requirements

- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

- This PR updates radio buttons found under vitals and biometrics to vertical tabs to match the designs

## Screenshots

Vitals and Biometrics Tabs

## Related Issue
Issue [MF-763](https://issues.openmrs.org/browse/MF-763)

### CODE REVIEW

File: packages/esm-patient-biometrics-app/src/biometrics/biometrics-chart.component.scss [hunk 1]
```diff
@@ -41,8 +41,47 @@
...
+
+.tab:hover {
+  border-bottom: none !important;
+}
+
+.tab > button:first {
+  text-indent: 5rem;
+}
+
+.tab>button,
```
Comment (FlorianRappl) [inline packages/esm-patient-biometrics-app/src/biometrics/biometrics-chart.component.scss:61]: Can we prettify this? Usually there are spaces here (`.tab > button`), see below. Consistency would be appreciated.

Same applies for many of the other lines, see e.g., line 80.

File: packages/esm-patient-vitals-app/src/vitals/vitals-chart.component.scss [hunk 2]
```diff
@@ -39,8 +39,47 @@
...
+
+.tab:hover {
+  border-bottom: none !important;
+}
+
+.tab > button:first {
+  text-indent: 5rem;
+}
+
+.tab > button,
```
Comment (FlorianRappl) [inline packages/esm-patient-vitals-app/src/vitals/vitals-chart.component.scss:59]: This file looks very similar...

Sure we cannot combine things here?

**Review by FlorianRappl — COMMENTED** [review]
I think this is nearly done. God job.

File: packages/esm-patient-biometrics-app/src/biometrics/biometrics-chart.component.scss (hunk 1 above)
Comment (maureennduta) [reply packages/esm-patient-biometrics-app/src/biometrics/biometrics-chart.component.scss]: Updated

File: packages/esm-patient-vitals-app/src/vitals/vitals-chart.component.scss (hunk 2 above)
Comment (maureennduta) [reply packages/esm-patient-vitals-app/src/vitals/vitals-chart.component.scss:59]: Yes these two files are similar. Vitals and Biometrics are generally similar. I think combining them can be a future enhancement?

Comment (FlorianRappl) [reply packages/esm-patient-vitals-app/src/vitals/vitals-chart.component.scss:59]: Alright let's do it like that.

**Review by FlorianRappl — APPROVED** [review]
Works for me!

**Pull request merged**